packages/store/src/patterns/patternMatchers.js

```diff
@@ -181,8 +289,11 @@ const makePatternKit = () => {
     // essentially identical.
     const checkIt = child => checkPattern(child, check);
```

Comment:
this should be moved to lin 305 so it's only allocated if used.

Reply:
Can't. It's used in two cases of the `switch` so needs to happen before the switch, or redundantly in each branch of the switch. If expensive. duplicating for each branch would be worth it. But this is only allocating the closure.

Resolving.